polus-imagej-filter-dog-plugin/README.md

```diff
@@ -0,0 +1,33 @@
+# ImageJ filter dog
+
+This WIPP plugin was automatically generated by a utility that searches for
+ImageJ plugins and generates code to run them. For more information on what this
+plugin does, contact one of the authors: Nick Schaub (nick.schaub@nih.gov) or 
+Anjali Taneja (anjali.taneja@axleinfo.com).
```

Comment:
You should update the README to include yourself as an author and remove Anjali's Axle email. Maybe include a link to her github profile.

Reply:
This section has been updated per your request to authors.